packages/provider/src/providers/near.ts

```diff
@@ -0,0 +1,13 @@
+import { ProviderProxy, RawProvider } from '../types'
+import { Providers } from '../enums'
+import { BaseNear } from './base-near'
+
+export class NearProvider extends BaseNear implements ProviderProxy {
+  constructor(provider: RawProvider) {
+    super(provider)
+  }
+
+  static get providerType(): Providers {
+    return Providers.Near
+  }
+}
```

Comment:
no need to do it with near provider coz u will have only one implementation

Reply:
fixed